Base/usr/share/man/man2/mount.md

```diff
@@ -21,9 +21,9 @@ int mount(const char* source, const char* target, const char* fs_type, int flags
 * `DevPtsFS` (or `devpts`): The pseudoterminal pseudo-filesystem (normally mounted at `/dev/pts`).
 * `TmpFS` (or `tmp`): A non-persistent filesystem that stores all its data in RAM. An instance of this filesystem is normally mounted at `/tmp`.
 
-For Ext2FS, `source` must be a path to a block device storing the filesystem contents. All
-the other filesystems ignore the `source` argument (by convention, it should have the same
-value as `fs_type`).
+For Ext2FS, `source_fd` must be a file descriptor of a file which stores the
+filesystem contents. All the other filesystems ignore the `source` argument (by
+convention, it should have the same value as `fs_type`).
```

Comment:
"For Ext2FS, `source_fd` must refer to an open file descriptor to a file containing the filesystem image. This may be a device file or any other seekable file. All the other filesystems ignore the `source_fd` — you can even pass an invalid file descriptor such as -1."

Reply:
Also, don't forget to update the "Bind mounts" and "Errors" sections below.